2-mandatory/1-weather-report.js

```diff
@@ -11,9 +11,26 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
+
+
+
+ function getTemperatureReport(cities) {
+
+let news = []
+
+for (let i = 0 ; i < cities.length; i++ ){
+
+let result  = temperatureService(cities[i])
+
+news.push (`The temperature in ${cities[i]} is ${result} degrees`)
+
+}return news;
+ }
```

Comment:
Great job! I believe the .map method could also be used in this scenario.

Reply:
> Great job! I believe the .map method could also be used in this scenario.

I think so